lib/db.js

```diff
@@ -84,7 +103,93 @@ class LimitDBRedis extends EventEmitter {
     });
   }
 
+  #setupPing() {
+    this.failedPings = 0;
+    this.redis.on('ready', () => this.#startPing())  
+  }
+
+  #startPing() {
+    if (this.keepPingRunning) {
```

Comment:
Perhaps this if is the other way around?
```suggestion
    if (!this.keepPingRunning) {
```

Reply:
Not needed anymore since I'm deprecating the keepPingRunning